### PR TITLE
New 'Update your Runtime' notification

### PR DESCRIPTION
**Pull request by akarukappadath**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

File: src/components/ClusterManager.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ export default class ClusterManager extends PureComponent {
       setDynamic(sessionStorage, `notifiedOutdatedCluster${cluster.id}`, Date.now())
       notify('warn', 'Please Update Your Runtime', {
         message: h(Fragment, [
-          p(['On Sunday Oct 20th at 10am, we are introducing important updates to Terra, which are not compatible with the older notebook runtime in this workspace. After this date, you will no longer be able to save new changes to notebooks in one of these older runtimes.']),
```
Comment (ehanna4) [inline src/components/ClusterManager.js:170]: Should this date include the year and time zone? i.e. 'Sunday Oct 20th, 2020 at 10am EST' ? (Also if this time is based on the TerraUI deployment cycle we actually release at 11am now (due to daylights savings time switch and we didn't adjust it))

Comment (akarukappadath) [reply src/components/ClusterManager.js:170]: I don't think it needs to be quite that specific, but I can update to "In October of 2019," if you think that would be helpful to users.

Comment (zarsky-broad) [reply src/components/ClusterManager.js:170]: I think @ehanna4 was looking at the old removed text and not the replacement...?

Comment (ehanna4) [reply src/components/ClusterManager.js:170]: 🤦‍♀️yup sorry, never mind on my last comment

**Review by ehanna4 — APPROVED** [review]
LGTM, tested in UI with the changes

**Pull request merged**